packages/portfolio-contract/test/contract-setup.ts

```diff
@@ -65,7 +61,7 @@ const deploy = async (t: ExecutionContext) => {
     {}, // terms
     {
       ...common.commonPrivateArgs,
-      contracts: contractAddressesMock,
+      axelarConfig: axelarConfigMock,
```

Comment:
Shouldn't the contract address stuff still be separate from axelar config? Contract addresses are not decided by axelar, right?